src/dymaptic.GeoBlazor.Core/Components/Widgets/BasemapLayerListWidget.cs

```diff
@@ -25,6 +25,7 @@ public class BasemapLayerListWidget : Widget
     /// </summary>
     [Parameter]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    [Obsolete("Use Icon instead")]
```

Comment:
is this removed in the other PR, can we just remove this here?

Reply:
This was originally targeted to update `develop` rather than `v3.0.0`, hence the difference in leaving the `Obsolete` tags. However, after thinking it through, I decided to go ahead and move this with the other new PRs to target v3, and remove the `IconClass`.